app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -184,7 +184,6 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
 
   if (error) {
     return <Alert variant={AlertVariants.Error} message={error} />;
-
```

Comment:
nit: revert this. This will mark the file as changed, but it really wasn't.

Reply:
Will revert the file